conf/sample.ini

```diff
@@ -488,6 +488,15 @@
 # Read the auth proxy docs for details on what the setting below enables
 ;enable_login_token = false
 
+#################################### Auth JWT ##########################
+[auth.jwt]
```

Comment:
All possible JWT settings should be included here for consistency, see `[auth.ldap]` or `[auth.proxy]` for examples. Seems like `jwk_set_file`, `key_file` and `expected_claims` is missing.